.drone.yml

```diff
@@ -28,101 +36,143 @@ steps:
     image: autonomy/build-container:latest
     pull: always
     environment:
-      BUILDKIT_HOST: tcp://buildkitd.ci.svc:1234
+      BUILDKIT_HOST: unix:///run/buildkit/buildkitd.sock
       BINDIR: /usr/local/bin
     commands:
       - make lint
+    volumes:
+      - name: buildkitsock
+        path: /run/buildkit
     depends_on:
       - fetch
 
+  # We'll use this step to load the hyperkube image to save
```

Comment:
not a fan of this... this is getting more and more down the path of managing dependencies on our own, which we already got bit by with the rootfs/test.

Reply:
This is to speed up the integration test, not for any other reason. We can kick off the docker pull in parallel with the build steps so when we get to running the integration test we don't need to spend the time waiting on docker pull.

Reply:
Yeah I get why, just adds in yet another place we need to be mindful of when upgrading Kubernetes. I'd rather not have this step. Breaking out every piece is already diverging from the original design to have dependencies managed in on place. I'd rather not go further down that path.